src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift

```diff
@@ -788,18 +794,15 @@ class HealthCertificateService: HealthCertificateServiceServable {
 		}
 	}
 
-	private func updateTimeBasedValidityStates() {
+	private func updateValidityStates() {
 		healthCertifiedPersons.forEach { healthCertifiedPerson in
 			healthCertifiedPerson.healthCertificates.forEach { healthCertificate in
-				updateTimeBasedValidityState(for: healthCertificate)
+				updateValidityState(for: healthCertificate, person: healthCertifiedPerson)
 			}
 		}
 	}
 
-	private func updateTimeBasedValidityState(for healthCertificate: HealthCertificate) {
-		guard healthCertificate.validityState != .invalid && healthCertificate.validityState != .blocked else {
-			return
-		}
+	private func updateTimeBasedValidityState(for healthCertificate: HealthCertificate, person: HealthCertifiedPerson) {
```

Comment:
Looks like the additional person parameter is not used

Reply:
done

Reply:
@nickguendling